SA3/qa_web/models.py

```diff
@@ -3,12 +3,52 @@
 
 
 class Post(models.Model):
-    content = models.TextField()
-    owner = models.ForeignKey(User, on_delete=models.SET_NULL, null=True)
+    """
+    Parent meta class that describes a post as a publication on the website with certain content
+    created by a certain owner
+    This will be used for inheritance
+    """
+    content = models.TextField(null=True)
+    owner = models.ForeignKey(User, null= True, on_delete=models.SET_NULL)
```

Comment:
The Django document raises some concerns about 'reference User directly'. It's suggested that using it this way may be better:
owner = models.ForeignKey(
        settings.AUTH_USER_MODEL,
or use get_user_model()
As [details](https://docs.djangoproject.com/en/2.0/topics/auth/customizing/).

Reply:
Good thing you pointed this out, if we want to plan ahead for #32 where we will probably have to define a custom User, this would be the right thing to do.

Reply:
Great observation, I did not consider that beforehand. First of all, since we are using only one app in our project, specifying the `AUTH_USER_MODEL` will be rather unnecessary, as `User` is already the _else_ condition for its use. 

![image](https://user-images.githubusercontent.com/29107916/36077496-a162c888-0f39-11e8-8e5d-ea9ffa2f08f6.png)

Furthermore, considering using versatile code for multiple apps should be defined at the start of the project.

![image](https://user-images.githubusercontent.com/29107916/36077475-5aae2d92-0f39-11e8-86fd-e912169e4b81.png)

Reply:
Good comment. Let's leave it.

Reply:
If we want the change the User profile in a further issue (#32), I suggest we reconsider this right away as doing so requires a correct initial migration. Explanation [here](https://docs.djangoproject.com/en/2.0/topics/auth/customizing/#changing-to-a-custom-user-model-mid-project)

Reply:
[Here ](https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html)is an article about extending django User model. If we don't want to change authentication process in #32 , we could just use option 2 or option 4.